app/models/concerns/cable_ready/updatable.rb

```diff
@@ -43,9 +43,9 @@ def cable_ready_collections
         @cable_ready_collections ||= CollectionsRegistry.new
       end
 
-      def cable_ready_update_collection(resource, name)
+      def cable_ready_update_collection(resource, name, model)
         identifier = resource.to_global_id.to_s + ":" + name.to_s
-        ActionCable.server.broadcast(identifier, {})
+        ActionCable.server.broadcast(identifier, {changed: model.previous_changes.keys})
```

Comment:
I think this will need a `respond_to? :previous_changes` guard to account for POROs, as exemplified by the need to add that method to the test dummy PORO

Reply:
![TIL](https://media.giphy.com/media/UvWllmm27ZaNbRpv2n/giphy.gif)